server/views/formBuilder.njk

```diff
@@ -0,0 +1,5 @@
+{% macro formBuilder(params) %}
+    {% if params.inputTemplate == 'placeOfResidence' %}
+        {% include 'partials/additionalConditionsInputs/placeOfResidence.njk' %}
+    {% endif %}
+{% endmacro %}
```

Comment:
Would we add an include for each group or condition? Not sure if the pages are being split by group or individual condition yet.

Reply:
The way the prototype currently looks, we would have an `include` here for each nested set of inputs. However again I expect this might change as we iterate